micro-demos/replicasets/rs.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: extensions/v1beta1
```

Comment:
I got feed back that yaml file is more easy to read than running `kubectl run` with many flags, so I made a yaml configuration file for replicasets.